src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -93,6 +93,7 @@ public async Task<ActionResult> GetMessageBoxInstanceList(int instanceOwnerParty
         /// <param name="instanceGuid">the instance guid</param>
         /// <param name="language"> language id en, nb, nn-NO"</param>
         /// <returns>list of instances</returns>
+        [Authorize(Policy = "InstanceRead")]
```

Comment:
Should we use a constant value instead of a string?
In case of a policy name change or something similar?
Same for all [Authorize(Policy = "...")]

Reply:
Yeah, sounds good. I fix.